chain/ethereum/src/chain.rs

```diff
@@ -368,6 +362,26 @@ impl TriggersAdapterTrait<Chain> for TriggersAdapter {
             }))
         }))
     }
+
+    async fn parent_ptr(&self, block: &BlockPtr) -> Result<BlockPtr, Error> {
+        use futures::stream::Stream;
+        use graph::prelude::LightEthereumBlockExt;
+
+        let blocks = self
+            .eth_adapter
+            .load_blocks(
+                self.logger.cheap_clone(),
+                self.chain_store.cheap_clone(),
+                HashSet::from_iter(Some(block.hash_as_h256())),
+            )
+            .collect()
+            .compat()
+            .await?;
+        assert_eq!(blocks.len(), 1);
+        Ok(blocks[0]
+            .parent_ptr()
+            .expect("genesis block cannot be reverted"))
```

Comment:
It's just that the method is called `parent_ptr` - maybe add a comment to it that it will never be called for the genesis block (and is therefore ok to panic in that case) and have a comment by the `expect` that it's only used for reverts

Reply:
I've added the comments.